### PR TITLE
local.conf.sample: elaborate TCMODE:sokol-flex comment

### PR DESCRIPTION
**Pull request by hashraf91**
JIRA: SB-23194

### CODE REVIEW

File: conf/local.conf.sample [hunk 1]
```diff
@@ -12,7 +12,8 @@ MACHINE ??= "qemux86"
 # where FLEXDIR is the path to your Sokol Flex OS workspace
 #TOOLCHAINS_PATH ?= "${FLEXDIR}/toolchains"
 
-# Use the oe/yocto-built toolchain rather than the external
+# Uncomment to use the oe/yocto-built toolchain rather than the external (e.g.
+# CodeBench toolchain)
```
Comment (sumaiyyafarooq) [inline conf/local.conf.sample]: @Noor-Ahsan  Should it be CodeBench or Sourcery?

Comment (Noor-Ahsan) [reply conf/local.conf.sample]: @sumaiyyafarooq I checked the reference manual and it does not talk about sourcery toolchain. we have following
meta-sourcery Contains files that support the Sourcery CodeBench
toolchain.

So lets use "external Sourcery CodeBench toolchain". Thoughts?

Comment (sumaiyyafarooq) [reply conf/local.conf.sample]: It is fine.

Comment (hashraf91) [reply conf/local.conf.sample]: Updated.

**Pull request merged**